src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return String.valueOf(color);
     }
```

Comment:
You should avoid creating a new instance of Random every time the method is called. Instead, you can create a single instance of Random as a class-level variable and reuse it.